tests/prepare.mjs

```diff
@@ -60,10 +60,12 @@ const promises = fixtures.map((fixture) =>
     })
     await Promise.all(publishDirectories.map((dir) => rm(dir, { recursive: true, force: true })))
 
-    if (NEXT_VERSION !== 'latest') {
-      await setNextVersionInFixture(cwd, NEXT_VERSION, {
-        logPrefix: `[${fixture}] `,
-      })
```

Comment:
Note - the similar check is now done inside `setNextVersionInFixture` to avoid doing any actual changes for `latest`, but still allow to figure out if NEXT_VERSION satisfies potential fixture constraints like those:

https://github.com/opennextjs/opennextjs-netlify/blob/5e2813227aa439fc39bad993c51329833f502ea1/tests/fixtures/ppr/package.json#L15-L19

https://github.com/opennextjs/opennextjs-netlify/blob/5e2813227aa439fc39bad993c51329833f502ea1/tests/fixtures/middleware-node-runtime-specific/package.json#L15-L19

Note - those are not "dependencies" - those our own definitions that control which next versions are applicable for this fixture (for example for node middleware fixtures, we only allow next versions that actually support that, etc)